data/json/items/tool/pets.json

```
@@ -106,15 +106,9 @@
     "color": "dark_gray",
     "charged_qualities": [ [ "SHEAR", 3 ] ],
     "flags": [ "WATER_BREAK", "ALLOWS_BODY_BLOCK", "FRAGILE_MELEE" ],
-    "charges_per_use": 25,
-    "pocket_data": [
-      {
-        "pocket_type": "MAGAZINE_WELL",
-        "rigid": true,
-        "flag_restriction": [ "BATTERY_ULTRA_LIGHT" ],
-        "default_magazine": "light_minus_battery_cell"
-      }
-    ],
+    "use_action": { "type": "link_up", "cable_length": 5 },
+    "charges_per_use": 1,
```

Shearing is pretty time consuming activity, and with 850 w of power draw, so something like using it for 5 minutes would cost about 250 kj of energy
```suggestion
    "charges_per_use": 250,
```